scripts/operations_tests/operations_tests.py

```diff
@@ -31,8 +31,8 @@
 from mantidimaging.core.io.loader import loader  # noqa: E402
 from mantidimaging.core.operations.loader import load_filter_packages  # noqa: E402
 
-LOAD_SAMPLE = (Path.home() / "mantidimaging-data" / "ISIS" / "IMAT" / "IMAT00010675" / "Tomo" /
-               "IMAT_Flower_Tomo_000000.tif")
+LOAD_SAMPLE = Path("C:/Users/44770/Downloads/mantidimaging-data-small "
+                   "(2)/mantidimaging-data-small/ISIS/IMAT/IMAT00010675/Tomo/IMAT_Flower_Tomo_000000.tif")
```

Comment:
Can you remove this change from the PR.

(Making this configurable might be a thing to consider in your improvements for the script.)